pkg/cluster/install.go

```diff
@@ -76,6 +76,7 @@ func (m *manager) adminUpdate() []steps.Step {
 			steps.Action(m.configureAPIServerCertificate),
 			steps.Action(m.configureIngressCertificate),
 			steps.Action(m.populateRegistryStorageAccountName),
+			steps.Action(m.EnsureMTUSize),
```

Comment:
```suggestion
			steps.Action(m.ensureMTUSize),
```

Should be defined and called only within `pkg/cluster`